--- usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js.orig	2009-09-22 23:51:00.000000000 +0930
+++ usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js	2009-09-22 23:57:24.000000000 +0930
@@ -29,6 +29,11 @@
 
 function setUpFirstUse() {
 Mojo.Log.info("---------------- setupFirstUse() --------------------");
+
+	setupSimulator();
+	isDevice = true;
+	return;
+
 	this.firstUseDepot = new Mojo.Depot(stateEngine, dbSetUpSuccess, dbSetupFailure);	
 	isInitialized = true;
 	attemptCount = 0;
--- usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js.orig	2009-09-23 00:01:12.000000000 +0930
+++ usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js	2009-09-23 00:10:58.000000000 +0930
@@ -137,10 +137,12 @@
 		Mojo.Log.info("------------ tapHandler show pop up--------- %j", near);
 		Mojo.Log.info("------------ tapHandler show pop up---------this.controller", this.controller);
   		var labels = [{label: $L('Emergency call'), command: 'emergency-call'}, 
+			{label: $L('Enable Wifi'), command: 'enable-wifi'},
 			{label: $L('Cancel'), command: 'cancel'}
 			];
 		if(FirstUseUtil.customerCareNumber && FirstUseUtil.customerCareNumber.length > 0) {
 			labels = [{label: $L('Emergency call'), command: 'emergency-call'}, 
+				{label: $L('Enable Wifi'), command: 'enable-wifi'},
 				{label: $L('Call customer service'), command: 'customer-care'},
 				{label: $L('Cancel'), command: 'cancel'}
 			];
@@ -155,6 +157,14 @@
   	popupChoose: function(value) {
 		if(value == "emergency-call") {
 			AppAssistant.accountService.makeEmergencyCall(FirstUseUtil.emergencyNumber);
+		} else if (value == "enable-wifi") {
+			return new Mojo.Service.Request("palm://com.palm.applicationManager/open", {
+				parameters: {
+				    id: 'com.palm.app.wifi',
+				    params: {
+				    }
+				}
+			    });
 		} else if (value == "customer-care") {
 			Mojo.Log.info("----------------- FirstUseUtil.carrierName ----------------", FirstUseUtil.carrierName());
 			AppAssistant.accountService.callCustomerCare(FirstUseUtil.customerCareNumber, FirstUseUtil.carrierName());
