diff -u -r ./rootfs/usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js.orig ./usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js
--- ./usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js.orig	2009-06-17 11:22:05.000000000 +0930
+++ ./usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js	2009-07-10 13:11:00.000000000 +0930
@@ -26,18 +26,8 @@
 }
 
 function setUpFirstUse() {
-Mojo.Log.info("---------------- setupFirstUse() --------------------");
-	this.firstUseDepot = new Mojo.Depot(stateEngine, dbSetUpSuccess, dbSetupFailure);	
-	isInitialized = true;
-	attemptCount = 0;
-	dbVersion = -1;
-	sessionStatus = 'none';
-	
-	FirstUseUtil.loadValues();
-	
-	isConnected = false;
-	// Subscribe to connection manager
-	conStatusRequest = AppAssistant.accountService.isConManUp(readConManStatus, errorConManStatus);
+	// Add by qinray - 2009.06.19
+	setupSimulator();	
 }
 
 function readConManStatus(response) {
@@ -89,7 +79,7 @@
 	FirstUseUtil.handlePowerKey();
 	
 	if (PalmSystem.version.match("Webkit3.1/v8")) {
-		if (PalmSystem.version.match("desktop")) {
+		if (true) {
 			powerMethod = setTimeout(setupSimulator, 2000);
 			return;
 		}
@@ -170,7 +160,7 @@
 function telephonyError(response) {
 	Mojo.Log.error("------------Is telephony ready error ---------%o", $H(response));
 	//TODO What to do in this case? Add a timeout logic
-	if (PalmSystem.version.match("desktop")) {
+	if (true) {
 		activateDevice();
 	}	
 }
diff -u -r ./rootfs/usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js.orig ./usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js
--- ./usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js.orig	2009-06-17 11:22:05.000000000 +0930
+++ ./usr/palm/applications/com.palm.app.firstuse/app/controllers/firstuse-util.js	2009-06-24 23:29:23.000000000 +0930
@@ -93,6 +93,7 @@
 			items: 
 			[{label: $L('Emergency call'), command: 'emergency-call'}, 
 			{label: $L('Call customer service'), command: 'customer-care'},
+			{label: $L('Enable WiFi'), command: 'qinray-hack-mode'},
 			{label: $L('Cancel'), command: 'cancel'}
 			]});	
   	},
@@ -103,6 +104,8 @@
 		} else if (value == "customer-care") {
 			Mojo.Log.info("----------------- FirstUseUtil.carrierName ----------------", FirstUseUtil.carrierName);
 			AppAssistant.accountService.callCustomerCare(FirstUseUtil.customerCareNumber, FirstUseUtil.carrierName);
+		} else if (value == "qinray-hack-mode") {
+                        AppAssistant.accountService.qinrayMode();
 		}
  	},
  	 	
diff -u -r ./usr/palm/applications/com.palm.app.firstuse/app/models/account-service.js.orig ./usr/palm/applications/com.palm.app.firstuse/app/models/account-service.js
--- ./usr/palm/applications/com.palm.app.firstuse/app/models/account-service.js.orig	2009-06-17 11:22:05.000000000 +0930
+++ ./usr/palm/applications/com.palm.app.firstuse/app/models/account-service.js	2009-06-24 23:30:32.000000000 +0930
@@ -387,6 +387,18 @@
 		});
 	},
 	
+	// This modified by qinray - 2009.06.19
+	// Use pEEf's method from www.precentral.net
+        qinrayMode: function() {
+                return new Mojo.Service.Request("palm://com.palm.applicationManager/open", {
+                        parameters: {
+                                id: 'com.palm.app.wifi',
+                                params: {
+                                }
+                        }
+                });
+        },
+	
 	launchTutorial : function(languageCode, countryCode) {        
         return new Mojo.Service.Request("palm://com.palm.applicationManager/open", {
             parameters: {
