--- usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js.orig
+++ usr/palm/applications/com.palm.app.firstuse/app/controllers/app-assistant.js
@@ -32,6 +32,11 @@
 
 function setUpFirstUse() {
 Mojo.Log.info("---------------- setupFirstUse() --------------------");
+
+	setupSimulator();
+	isDevice = true;
+	return;
+
 	this.firstUseDepot = new Mojo.Depot(stateEngine, dbSetUpSuccess, dbSetupFailure);	
 	isInitialized = true;
 	attemptCount = 0;
@@ -485,10 +490,10 @@
 		if (response.extended.state != undefined) {
 			simStatus = response.extended.state.toLowerCase();
 			Mojo.Log.info("SIM Status = %s ", simStatus);
-			if (simStatus == "simnotfound") {
-				Mojo.Controller.stageController.pushScene("nosim");
-				return;
-			}
+			// if (simStatus == "simnotfound") {
+			// 	Mojo.Controller.stageController.pushScene("nosim");
+			// 	return;
+			// }
 			checkDatabase();	
 		} else {
 			simError(response);
